packages/buttons/stories/2-Button.stories.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Buttons/Button',
+  component: Button
+} as Meta;
+
+export const Default: Story = ({
+  isDanger,
+  size,
+  isStretched,
+  isPrimary,
+  isBasic,
+  isLink,
+  isPill,
+  focusInset
+}) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <Button
+          isDanger={isDanger}
+          size={size}
+          isStretched={isStretched}
+          isPrimary={isPrimary}
+          isBasic={isBasic}
+          isLink={isLink}
+          isPill={isPill}
+          focusInset={focusInset}
+        >
+          Click
+        </Button>
+      </Col>
+    </Row>
+  </Grid>
+);
+
+Default.argTypes = {
+  isDanger: {
+    control: { type: 'boolean' }
+  },
+  size: {
+    control: { type: 'select', options: ['small', 'medium', 'large'] }
+  },
+  isStretched: {
+    control: { type: 'boolean' }
+  },
```

Comment:
Are the boolean types necessary due to the `undefined` variable? We might be able to just assign `false` to these in

```
Default.args = {};
```

Reply:
Hmm removing the highlighted code and adding the following results in the boolean control not rendering.

```jsx
Default.args = {
  isStretched: false
};
```

Reply:
From https://github.com/zendeskgarden/react-components/blob/main/packages/loaders/stories/5-Skeleton.stories.tsx it looks like these can be simplified to:

```js
isPrimary: {
  control: 'boolean'
}
```

Reply:
Updated to simplified.